packages/fxa-auth-server/test/local/payments/stripe.js

```diff
@@ -1298,6 +1298,43 @@ describe('StripeHelper', () => {
     });
   });
 
+  describe('allTaxRates', () => {
+    it('pulls a list of tax rates and caches it', async () => {
+      assert.lengthOf(await stripeHelper.allTaxRates(), 2);
+      assert(mockRedis.get.calledOnce);
+
+      assert.lengthOf(await stripeHelper.allTaxRates(), 2);
+      assert(mockRedis.get.calledTwice);
+      assert(mockRedis.set.calledOnce);
+
+      // Assert that a TTL was set for this cache entry
+      assert.deepEqual(mockRedis.set.args[0][2], [
+        'EX',
+        mockConfig.subhub.plansCacheTtlSeconds,
+      ]);
+
+      assert(stripeHelper.stripe.taxRates.list.calledOnce);
+
+      assert.deepEqual(
+        await stripeHelper.allTaxRates(),
+        JSON.parse(await mockRedis.get('listActiveTaxRates'))
+      );
+    });
+  });
+
+  describe('taxRateByCountryCode', () => {
+    it('locates an existing tax rate', async () => {
+      const result = await stripeHelper.taxRateByCountryCode('FR');
+      assert.isDefined(result);
+      assert.deepEqual(result, taxRateFr);
+    });
+
+    it('returns undefined for unknown tax rates', async () => {
+      const result = await stripeHelper.taxRateByCountryCode('GA');
+      assert.isUndefined(result);
+    });
```

Comment:
These functions aren't used yet in the code, but I'm curious if this occurs in the wild, would we want to throw an error or log?

Reply:
The caller probably should log an issue if its an EU country with no tax rate found, but this function should be fine as is.